#define PARAM_NAME_TREES_FOLDER "trees_folder"
#define PARAM_NAME_ENABLE_COUT_LOG "enable_cout_log"
#define PARAM_NAME_ENABLE_MINITRACE_LOG "enable_minitrace_log"
#define PARAM_NAME_ENABLE_ROSTOPIC_LOG "enable_rostopic_log"
#define PARAM_NAME_ENABLE_FILE_LOG "enable_file_log"
#define PARAM_NAME_ENABLE_GROOT_LOG "enable_groot_log"
#define PARAM_NAME_TREE_NAME "tree_name"
#define PARAM_NAME_TREE_FILE "tree_file"
#define PARAM_NAME_TREE_UID "tree_uid"
#define PARAM_NAME_TREE_DEBUG "tree_debug"
#define PARAM_NAME_TREE_AUTORESTART "tree_auto_restart"
#define PARAM_NAME_TREE_SERVER_PORT "server_port"
#define PARAM_NAME_TREE_PUBLISHER_PORT "publisher_port"
#define PARAM_NAME_TREE_LOG_FOLDER "log_folder"
#define PARAM_NAME_TREE_BB_INIT "bb_init"
#define PARAM_NAME_TREE_ROS_PLUGINS_DIR "ros_plugins_dir"   
#define PARAM_NAME_TREE_LOOP_RATE "loop_rate"
